rir/src/interpreter/instance.h

```diff
@@ -172,14 +172,19 @@ class Locals final {
   private:
     R_bcstack_t* base;
     unsigned localsCount;
+    bool existingLocals;
```

Comment:
Do not fully get this bool? Does it have any relation with inlining? When are the locals already reserved on the stack?

Reply:
for inlinedContexts we recursively call the interpreter function, but we just continue executing as before. in particular we continue using the same locals and the stack. the additional interpreter execution is just to sneak a context trampoline in between, to bind the setjmps.